### PR TITLE
XW-1753 | set content lang

### PR DESCRIPTION
**Pull request by tomasznapieralski**
content is set on:
- article page
- curated content
- discussions
- RWA

### CODE REVIEW

File: server/app/facets/mediawiki-page.js [hunk 1]
```diff
@@ -144,6 +144,10 @@ function handleResponse(request, reply, data, allowCache = true, code = 200) {
 	// @todo XW-596 we shouldn't rely on side effects of this function
 	Tracking.handleResponse(result, request);
 
+	if (result.wikiVariables && result.wikiVariables.language && result.wikiVariables.language.content) {
```
Comment (rogatty) [inline server/app/facets/mediawiki-page.js]: How about extracting it to an util?

**Comment by Warkot** [comment]
LGTM

File: server/app/facets/mediawiki-page.js (hunk 1 above)
Comment (Warkot) [inline server/app/facets/mediawiki-page.js]: I was also thinking about it. We're talking about it now. Wouldn't it make code less readable actually?

Comment (Warkot) [inline server/app/facets/mediawiki-page.js]: Talked offline—let's DRY it! :smi

**Comment by rogatty** [comment]
LGTM

**Pull request merged**